classes/issue/maps/Schema.inc.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+namespace APP\issue\maps;
+
+use APP\core\Application;
+use APP\core\Services;
+use APP\facades\Repo;
+use APP\issue\Issue;
+use APP\issue\IssueGalleyDAO;
+use APP\journal\SectionDAO;
+use Illuminate\Support\Enumerable;
+use PKP\context\Context;
+use PKP\core\PKPRequest;
+use PKP\db\DAORegistry;
+use PKP\services\PKPSchemaService;
+use UserGroupDAO;
+
+class Schema extends \PKP\core\maps\Schema
+{
+    /** @var Enumerable */
+    public $collection;
+
+    /** @var string */
+    public $schema = PKPSchemaService::SCHEMA_ISSUE;
+
+    public function __construct(PKPRequest $request, Context $context, PKPSchemaService $schemaService)
+    {
+        parent::__construct($request, $context, $schemaService);
+    }
```

Comment:
You can remove this constructor. The parent will be called automatically if it is missing.